cxx_flag_overrides.cmake

```diff
@@ -4,7 +4,7 @@ set(CMAKE_CXX_FLAGS_RELEASE_INIT "-O3 -DNDEBUG")
 set(CMAKE_CXX_FLAGS_RELWITHDEBINFO_INIT "-O2 -ggdb3 -fno-omit-frame-pointer -fsanitize=address -fsanitize=null")
 
 if (CMAKE_CXX_COMPILER_ID MATCHES "GNU")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-array-bounds -Wno-stringop-overflow")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-array-bounds -Wno-stringop-overflow -pipe")
```

Comment:
Did we do proper tests with this option though ? Because last time I thought it wasn't faster than without it.

Reply:
Nope, no question. just "add it it's faster" and I added it :laughing:

Reply:
Imagine if we could use shared_ptr :exploding_head: